SendGrid/SendGridMail/Transport/Web.cs

```diff
@@ -87,7 +87,7 @@ public void Deliver(ISendGrid message)
 			AttachFormParams(message, content);
 			AttachFiles(message, content);
 			var response = await client.PostAsync(Endpoint + ".xml", content);
-			await CheckForErrorsAsync(response);
+            await ErrorChecker.CheckForErrorsAsync(response);

```

Comment:
You have to await it because it's dependent on the response and because the response is already awaited, it has to be awaited too.